python/ray/train/tensorflow/tensorflow_predictor.py

```diff
@@ -225,9 +225,9 @@ def predict(
     def _arrays_to_tensors(
         self,
         numpy_arrays: Union[np.ndarray, Dict[str, np.ndarray]],
-        dtypes: Union[tf.dtypes.DType, Dict[str, tf.dtypes.DType]],
+        dtype: Optional[Union[tf.dtypes.DType, Dict[str, tf.dtypes.DType]]],
```

Comment:
@amogkam I changed the name of the `dtypes` parameter to `dtype`, so that the signature of matches `DLPredictor._arrays_to_tensors`. I made a similar change to `TorchPredictor`.

Doing it in this PR because I don't want the overhead of creating/reviewing/merging a separate PR.